test/spec/nativeTrackerManager_spec.js

```diff
@@ -89,6 +95,58 @@ describe('nativeTrackerManager', function () {
       expect(trimPort(postMessageTargetDomain)).to.equal(tagData.pubUrl);
     });
 
+    /*it('should verify the postMessage for impression trackers was executed', function() {
```

Comment:
what is the reasoning behind this being commented out?

Reply:
I forgot to remove this test. Originally the new rendering code lived within trackerManager code but it was requested to live in a new class. This was mirrored in the nativeRenderManager_spec test.